.github/workflows/ci.yaml

```diff
@@ -163,7 +159,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, 3.10-dev]
```

Comment:
3.10-dev simply can't install all dependencies when using poetry, so let's wait until 3.10 is released properly before attempting to fix that, or if people are using the 3.10-rc maybe they will create reproduction steps and raise issues.

Reply:
I added 3.10 in the meta-information in the ``setup.py`` it's probably best to remove it then.